### PR TITLE
xGov 183: Choice Coin Compliance

### PR DESCRIPTION
**Pull request by ChoiceCoin**

### CODE REVIEW

**Comment by SudoWeezy** [comment]
Hey, based on [ARC-34](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0034.md), you can have up to one milestone per proposal. It will not be allowed to move to vote with this current version.

**Comment by ChoiceCoin** [comment]
> Hey, based on [ARC-34](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0034.md), you can have up to one milestone per proposal. It will not be allowed to move to vote with this current version.

Thanks. I edited the amount requested to only include the upfront payment and the first milestone award. 

https://github.com/ChoiceCoin/xGov/blob/main/Proposals/xGov%20183:%20Choice%20Coin%20Compliance

Please let me know if this is not sufficient for approval in compliance with ARC-34. Thank you.

File: Proposals/xGov 183: Choice Coin Compliance [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+Team Member Responsible: Brian Haney
+Grant Payment: 14,000 ALGO
+
+Milestone 1: Database Generation
+Deliverables: Database, GitHub OSS Repository, Milestone Report
+Schedule: 4 weeks
+Team Member Responsible: Brian Haney and David Kazeem
+Grant Payment: 39,000 ALGO
+
+Milestone 2: Research and Development
```
Comment (SudoWeezy) [inline Proposals/xGov 183: Choice Coin Compliance:108]: Milestones 2, 3, and 4 should go to "Future blueprint".
You asked for 53000 Algo; you should regroup milestones (0 & 1). As I said in my last comment, you can only have one milestone.

Comment (ChoiceCoin) [reply Proposals/xGov 183: Choice Coin Compliance:112]: I just updated it. I understand there is only one milestone. But, it was not clear what your required protocol was for listing out the milestones. I amended the proposal per your request.

**Comment by Bhaney44** [comment]
I am happy to provide the deliverables for our project. Thank you for the opportunity and your support!

Deliverable link: https://github.com/ChoiceCoin/algo-compliance

**Pull request merged**